prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -297,7 +302,18 @@ public override async Task<bool> ProcessRule(Enrollee enrollee)
                     enrollee.AddReasonToCurrentStatus(StatusReasonType.PossiblePaperEnrolmentMatch, $"Birthdate matches enrolment(s): {paperEnrolleeIdsAsString}");
                     return false;
                 }
+
+                // First enrolment: check if the related paper enrolment is flagged for AlwaysManual
+                // If so, link enrolments and mark BCSC enrolment as AlwaysManual too and send to manual enrolment
+                if (paperEnrollees.Any(pe => pe.Id == paperEnrolleeMatchId && pe.AlwaysManual))
+                {
+                    await _enrolleePaperSubmissionService.UpdateAlwaysManualAsync(enrollee.Id, true);
```

Comment:
why make/call a service method and not just do
```c#
enrollee.AlwaysManual = true;
```

Reply:
updated